paddle/scripts/travis/build_doc.sh

```diff
@@ -5,13 +5,6 @@ set -e
 mkdir -p $TRAVIS_BUILD_DIR/build
 cd $TRAVIS_BUILD_DIR/build
 
-# Compile paddle binaries first
-cmake .. -DCMAKE_BUILD_TYPE=Debug -DWITH_GPU=OFF -DWITH_DOC=OFF -DWITH_MKLDNN=OFF -DWITH_MKLML=OFF -DWITH_GOLANG=ON -DWITH_STYLE_CHECK=OFF
```

Comment:
Yes, `build_doc` Travis Job doesn't build twice,  and doesn't build the whole Paddle now.